src/Http/Controllers/FilepondController.php

```diff
@@ -60,13 +56,15 @@ public function upload(Request $request)
      * Takes the given encrypted filepath and deletes
      * it if it hasn't been tampered with
      *
-     * @param Request $request
+     * @param  Request $request
+     *
      * @return mixed
+     * @throws \Illuminate\Contracts\Filesystem\FileNotFoundException
      */
     public function delete(Request $request)
     {
         $filePath = $this->filepond->getPathFromServerId($request->getContent());
-        if(unlink($filePath)) {
+        if (unlink($filePath)) {
```

Comment:
Looks like I forgot to handle removing a file on the selected disk. I will get that fixed ASAP and then unmark this as Draft.